src/CMakeLists.txt

```diff
@@ -121,7 +122,7 @@ if (ENABLE_PFRING_SUPPORT)
     # Set path to manually compiled PF_RING
     set(PFRING_INCLUDE_DIRS "${PFRING_CUSTOM_INSTALL_PATH}/include")
 
-    find_library(PFRING_LIBRARIES NAMES pfring PATHS "${PFRING_CUSTOM_INSTALL_PATH}/lib" NO_DEFAULT_PATH)
+    find_library(PFRING_LIBRARIES NAMES pfring PATHS "${PFRING_CUSTOM_INSTALL_PATH}/lib")
```

Comment:
We also added "NO_DEFAULT_PATH" for a reason. Usually, distributions ship very old versions of libraries and we're shipping them separately to cover FNM's requirement about requires features.